addons/website_payment/static/src/snippets/s_donation/options.js

```diff
@@ -27,6 +27,10 @@ options.registry.Donation = options.Class.extend({
         if (!this.$target[0].dataset.descriptions) {
             this._updateDescriptions();
         }
+        const customButtonEl = this.$target[0].querySelector("#s_donation_amount_input");
+        if (customButtonEl) {
+            customButtonEl.style.maxWidth = "";
+        }
```

Comment:
Shouldn't this be in the destroy of the 000.js then?

Reply:
Having the "add" in `000.js` and the "remove" in `options.js` indeed lacks symmetry.
But in my opinion, that cleanup only needs to happen before saving the page - it can remain "as set" when navigating away. By putting it in `options.js` we avoid including that cleanup code in the assets served to visitors.

Reply:
You described it right, it seems conceptually unusual since it's only required because of the `disabledInEditableMode: false,` in the 000.JS.
I am ok with keeping it like that but at least please add a comment to link those 2 parts together, something like
`This is cleaning the max-width added in the 000.js which is also run in Edit Mode`
And maybe on the 000.js code `This will be cleaned in the options.js cleanForSave`.

But honestly:
> it can remain "as set" when navigating away.

Destroy is not called when navigating away, only when entering edit mode/cloning etc. That's where we usually do that, like hiding the popup/cookies bar before entering edit mode, reseting and neutralizing countdown etc etc

> we avoid including that cleanup code in the assets served to visitors.

It's only 3 lines of code and on top of that, unless I am missing something, this is how ht system is build, no? On destroy, we clear/clean/reset everything for the edition which is to be started.

Reply:
Ok, it will be better with everything in `000.js`. :+1:  I moved it accordingly.